compiler/circle-tensordump/src/Dump.cpp

```diff
@@ -228,6 +228,8 @@ std::vector<hsize_t> hdf5_dims_cast(const flatbuffers::Vector<T> *data,
       ret.resize(rank);
       for (uint32_t d = 0; d < rank; d++)
       {
+        if (dims->Get(d) < 0)
+          throw std::runtime_error("negative value cannot be casted to unsigned");
```

Comment:
This message is kinda weird.
```suggestion
          throw std::runtime_error("Dimensions shouldn't be negative");
```